### ISSUE TITLE
manage command errors

### ISSUE THREAD

**Issue by jedie** [edoburu/django-oscar-docdata#12]
`manage.py update_docdata_order` with django 1.11:
```
  File "django-oscar-docdata/oscar_docdata/management/commands/update_docdata_order.py", line 11, in <module>
    class Command(BaseCommand):
  File "django-oscar-docdata/oscar_docdata/management/commands/update_docdata_order.py", line 18, in Command
    ) + BaseCommand.option_list
AttributeError: type object 'BaseCommand' has no attribute 'option_list'
```

**Comment by jedie** [comment]
`manage.py expire_docdata_orders` with django 1.11:

```
  File "django-oscar-docdata/oscar_docdata/management/commands/expire_docdata_orders.py", line 4, in <module>
    from django.core.management.base import NoArgsCommand
ImportError: cannot import name 'NoArgsCommand'
```
`docdata_report` has the same error...

**Comment by mj026** [comment]
Your commits have been merged into master with #22
